modules/ROOT/pages/policy-mule4-detokenization.adoc

```diff
@@ -62,7 +73,8 @@ If you want to detokenize the social security number, you can use the following
 #[payload.socialSecurityNumber]
 --
 
-Now, every time the policy receives a request containing a `socialSecurityNumber` key in its top level, it will extract all the tokens that match the expression you indicated, it will establish a connection with the tokenization service, and finally it will replace the original detokenized values in your payload with the ones provided by the service. +
+Now, every time the policy receives a request containing a `socialSecurityNumber` key in its top level, it will extract all the tokens that match the expression you indicated, it will establish a connection with the tokenization service, and finally it will replace the original detokenized values in your payload with the ones provided by the service. 
```

Comment:
```suggestion
Using this selector, every time the policy receives a request containing a `socialSecurityNumber` key in its top level, it extracts all the tokens that match the expression you indicated, establishes a connection with the tokenization service, and replaces the tokenized values in your payload with the original values. 
```
Do we need to include "in its top level"? It seems a useless piece of info to the user, that just complicates the sentence.

Reply:
Once again, I have been editing this PR for term logic:

Tokenization replaces a value with a token, to protect that value.
Detokenization returns the tokenized value to its original value.

Some of your original text implies just the opposite, so please very carefully evaulate the suggested rewordings to ensure that I have not misunderstood and that your intended meaning is reflected.

Reply:
Done! Thanks!